### PR TITLE
feat(app): create core class

### PR DESCRIPTION
**Pull request by Muunatic**
add core class to increase efficiency the app structure by creating separate classes. this approach helps decrease complexity and makes it easier to maintain.

### CODE REVIEW

File: src/class/prsSynchronize.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                                        pull_number: this.context.payload.pull_request.number,
+                                        review_id: res.data[i].id,
+                                        message: "This review is stale and has been dismissed."
+                                    });
+                                    await this.context.octokit.pulls.requestReviewers({
+                                        owner: this.context.payload.repository.owner.login,
+                                        repo: this.context.payload.repository.name,
+                                        pull_number: this.context.payload.pull_request.number,
+                                        reviewers: [username]
+                                    });
```
Comment (Muunatic) [inline src/class/prsSynchronize.ts:68]: it appears that the changes made in PR #41 have been superseded by the recent changes in this line @HarunamiYaki

Comment (Muunatic) [inline src/class/prsSynchronize.ts:68]: it appears that the changes made in PR #41 have been superseded by the recent changes in this line @HarunamiYaki

**Review by Muunatic — COMMENTED** [review]
PRs #41 changes

File: src/class/prsSynchronize.ts (hunk 1 above)
Comment (HarunamiYaki) [reply src/class/prsSynchronize.ts:68]: Aight

**Review by HarunamiYaki — APPROVED** [review]
Sounds good to me, let's go ahead and merge it 👍

**Comment by Muunatic** [comment]
ready to merge

**Pull request merged**